changelog.json

```diff
@@ -1,5 +1,17 @@
 {
   "releases": {
+    "2.9.1-beta1": [
+      "[Added] Add Fluent Terminal shell support - #12305. Thanks @Idered!",
+      "[Fixed] History tab shows all commits as the user scrolls down - #12506 #3704",
+      "[Fixed] Show co-authors from undone commits - #12537",
+      "[Fixed] Change the diff gutter width based on number of lines in diff - #2102. Thanks @ADustyOldMuffin!",
+      "[Fixed] Enable 'Open in External Editor' for bat/cmd/sh/exe files - #6361. Thanks @AndreiMaga!",
+      "[Fixed] Green circle in progress dialog remains round when commits have long descriptions - #12594. Thanks @litetex!",
+      "[Fixed] Removed border of Repository and Branch foldouts in dark theme - #12478. Thanks @meJevin!",
```

Comment:
Not sure if these two are worth mentioning… what do you think @billygriffin ?

Reply:
I like them, especially for beta, just because they credit the people who contributed.